.github/actions/javascript/postTestBuildComment/postTestBuildComment.js

```diff
@@ -0,0 +1,70 @@
+const core = require('@actions/core');
```

Comment:
It would be great to see some automated tests created for this JS action, to give us some additional confidence as reviewers, but also more importantly to make it easier to adjust the comment or code in the future.

Reply:
I will take care of that next week, it was very busy time and I was not able to pick it up...

Reply:
As far as I can tell there are no tests for JS files used in GH actions so far. Do you have any idea @roryabraham where should that sit and whether it should be executed together with all other tests?

On the one hand it will make it convenient to just create new dir within `tests` and rely on `npm test` command to execute this test together with all other jest tests. I feel like they don't logically belong together, though.